CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
 # CHANGELOG
+## 3.4.0
+#### New Functionality
+
+- Can now talk to MarkLogic using OAuth token (MarkLogic Server version 11.2.0 or higher needed).
+- Can now submit optic update plans to /v1/rows/update (MarkLogic Server version 11.2.0 or higher needed).
+- Can now use PatchBuilder for patching documents (MarkLogic Server version 11.2.0 or higher needed).
```

Comment:
A user won't know what "PatchBuilder" refers to here, we need to give them some context. So: "Can now perform partial updates on documents via Optic using the new `patchBuilder` operator." And I always try to include code mentions in backwards tick marks so they're formatted in Markdown, making them stand out more for the reader.